src/Nri/Ui/PremiumCheckbox/V6.elm

```diff
@@ -32,7 +32,7 @@ import Nri.Ui.Pennant.V1 exposing (premiumFlag)
 view :
     { label : String
     , id : String
-    , selected : Checkbox.IsSelected
```

Comment:
FYI: I checked in our code (monolith and monorepo) and we never use this name directly, so it will not require an update.